mono/metadata/marshal.c

```diff
@@ -5187,6 +5187,8 @@ ves_icall_System_Runtime_InteropServices_Marshal_PtrToStringAnsi_len (const char
 	return mono_string_new_utf8_len (mono_domain_get (), ptr, len, error);
 }
 
+#if !ENABLE_NETCORE
```

Comment:
Does this #if make sense? these methods won't be linked on netcore and we already have a lot of non-netcore icall impls in this file.

Reply:
It is true, it is not needed, and has a downside. I’ll remove it.

Reply:
Without the ifdef, we will get build warnings about non-static functions without a prototype, no?

Reply:
Also, the ifdefs are a good reminder of code which is no longer needed on netcore.